src/migrations/20230711094214-add-potentially-stale-flag.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+        ALTER TABLE features ADD COLUMN IF NOT EXISTS potentially_stale boolean;
+        UPDATE features
```

Comment:
Do we need this now that we have scheduler?

Reply:
Ah, good point! Maybe we don't. 

However, when the events come into play, we probably do want to keep it. This migration won't cause any events to emit, but if you rely on the scheduler to do it, then that could potentially emit thousands of events because it marks all your old toggles (that you haven't cleaned up) as stale.

What do you think?

Reply:
Sure, good point.